packages/wrangler/e2e/dev.test.ts

```diff
@@ -1,5 +1,7 @@
 import crypto from "node:crypto";
+import * as nodeNet from "node:net";
 import path from "node:path";
+import { afterEach } from "node:test";
```

Comment:
This should be from vitest

Reply:
@Skye-31 Ah my bad, it should be changed now. Also fixed the formatting errors that caused tests to fail